bin/tass

```diff
@@ -57,10 +57,17 @@ when 'create'
   end
 
   config = initialize_environment(opts)
+  sfr_config = {}  # @TODO: remove this when SFR config becomes sane
 
   # check the verison of the yaml we are reading, and do the new hotness if we are running v2 or higher
   if newer_yaml_version?(config)
     # Determine if we are making a cloudformation request
+    if config[:stack][:spot_fleet]
```

Comment:
This is fine for he alpha, but this will break in the event we have things that aren't spot fleets that we stand up in new world tass